src/infra/main/template.yaml

```diff
@@ -996,27 +1044,49 @@ Resources:
         Owner: !Ref OwnerTagValue
         Source: !Ref SourceTagValue
 
-  # PrivateAPIGatewayAccessLogGroup:
-  #   Type: AWS::Logs::LogGroup
-  #   Properties:
-  #     LogGroupName: !Sub "/aws/apigateway/${PrivateRestApi}/${ApiStageName}"
-  #     RetentionInDays: 30
-  #     KmsKeyId: !GetAtt CloudWatchEncryptionKey.Arn
-  #     Tags:
-  #       - Key: Product
-  #         Value: !Ref ProductTagValue
-  #       - Key: System
-  #         Value: !Ref SystemTagValue
-  #       - Key: Environment
-  #         Value: !Ref Environment
-  #       - Key: Name
-  #         Value: PrivateApiGatewayAccessLogGroup
-  #       - Key: Owner
-  #         Value: !Ref OwnerTagValue
-  #       - Key: Source
-  #         Value: !Ref SourceTagValue
-  #       - Key: Name
-  #         Value: !Sub "/aws/apigateway/${PrivateRestApi}/${ApiStageName}"
+  PrivateAPIGatewayAccessLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub "/aws/apigateway/${PrivateRestApi}/${ApiStageName}"
+      RetentionInDays: 30
+      KmsKeyId: !GetAtt CloudWatchEncryptionKey.Arn
+      Tags:
+        - Key: Product
+          Value: !Ref ProductTagValue
+        - Key: System
+          Value: !Ref SystemTagValue
+        - Key: Environment
+          Value: !Ref Environment
+        - Key: Name
+          Value: PrivateApiGatewayAccessLogGroup
+        - Key: Owner
+          Value: !Ref OwnerTagValue
+        - Key: Source
+          Value: !Ref SourceTagValue
+        - Key: Name
+          Value: !Sub "/aws/apigateway/${PrivateRestApi}/${ApiStageName}"
+
+  PrivateAPIAccessLogGroupNameSSM:
```

Comment:
Do we need this resource?

Reply:
dont think so, passes linting and deploys without so took it out